src/Collector/Formatter.php

```diff
@@ -44,11 +44,11 @@ public function __construct(MessageFormatter $formatter, CurlCommandFormatter $c
     /**
      * Formats an exception.
      *
-     * @param Exception $exception
+     * @param \Throwable $exception
      *
      * @return string
      */
-    public function formatException(Exception $exception)
+    public function formatException(\Throwable $exception)
```

Comment:
This is not a BC break because the class is internal.

Reply:
even more, Throwable is wide variation of Exception, so no BC break will happen

Reply:
Yes. But that reason alone is still a BC break because the class is not final.